app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml

```diff
@@ -17,5 +17,4 @@
 <adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
     <background android:drawable="@drawable/ic_launcher_background"/>
     <foreground android:drawable="@drawable/ic_launcher_foreground"/>
-    <monochrome android:drawable="@drawable/ic_launcher_foreground" />
```

Comment:
`ic_launcher_round` still has the `monochrome` item being removed, so round icons don't have a monochrome version available.

Could we keep the `monochrome` item in `ic_launcher` and ic_launcher_round` (net result is those files are unchanged by this PR)

That avoids adding another overload file, and aligns with the approach here: https://developer.android.com/guide/practices/ui_guidelines/icon_design_adaptive#add_your_adaptive_icon_to_your_app

Reply:
Removed roundIcon from the manifest, as per documentation. Removed v33 overload and included in v26 instead.